src/Estimators/tests/RobotDynamicsEstimator/FrictionTorqueDynamicsTest.cpp

```diff
@@ -0,0 +1,104 @@
+﻿/**
+ * @file FrictionTorqueDynamicsTest.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#include <catch2/catch.hpp>
+
+#include <ConfigFolderPath.h>
+#include <iCubModels/iCubModels.h>
+#include <yarp/os/ResourceFinder.h>
+
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/ParametersHandler/YarpImplementation.h>
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+#include <BipedalLocomotion/System/VariablesHandler.h>
+
+#include <BipedalLocomotion/RobotDynamicsEstimator/FrictionTorqueDynamics.h>
+
+using namespace BipedalLocomotion::Estimators::RobotDynamicsEstimator;
+using namespace BipedalLocomotion::ParametersHandler;
+using namespace BipedalLocomotion::System;
+
+double random_double()
+{
+    return 1.0 * ((double)rand() - RAND_MAX / 2) / ((double)RAND_MAX);
+}
```

Comment:
``catch`` should have dedicated methods to generate a random double https://github.com/catchorg/Catch2/blob/0a0ebf50034831718ffc63c6d12abb1fa469d6e9/docs/generators.md

Also, it is suggested to fix the seed, to make sure that the test is repeatable (meaning that the value is no more random, but at least you avoid cases where some test fails once in a while)